src/main/resources/META-INF/rewrite/was-to-liberty.yml

```diff
@@ -134,3 +135,13 @@ recipeList:
       newValue: http://xmlns.jcp.org/xml/ns/javaee
       versionMatcher: 3.1+
       searchAllNamespaces: false
+---
+type: specs.openrewrite.org/v1beta/recipe
+name:  org.openrewrite.maven.liberty.AddOpenLibertyPluginDependency
+displayName: Add Liberty Maven plugin dependencies
+description: This recipe adds required Open liberty plugin dependencies to the pom.xml file.
+recipeList:
+  - org.openrewrite.maven.AddDependency:
+      groupId: io.openliberty.tools
+      artifactId: liberty-maven-plugin
+      version: 3.10.3
```

Comment:
Any project this recipe runs against would unconditionally get the `liberty-maven-plugin` as a `<dependency>`. That includes adding this multiple times to multi-module projects. [Elsewhere I see](https://openliberty.io/guides/maven-intro.html#creating-the-project-pom-file) the `liberty-maven-plugin` added as `<plugin>` instead of `<dependency>`. Could you explain your choice here?

Reply:
you are right ..fixed in the next commit